### ISSUE TITLE
ARM Windows version of win8.1 universal app does not work

### ISSUE THREAD

**Issue by vick-joingame** [MSOpenTech/cocos2d-x#153]
Hi.
If makes an ARM version of the sample project or dummy project (win8.1 universal) if fails to run on the ARM windows devices with error "HelloCpp.Windows.exe: 0xC0000135: Unable to Locate DLL". Win32 version works fine.

**Comment by stammen** [comment]
Which version of cocos2d-x are you using?

**Comment by vick-joingame** [comment]
2.2.6 from branch v.2.2.6-universal (MSOpenTech/cocos2d-x), also tried to use from cocos2dx repository. Looks like some DLLs (from ARM directory) from third-party-libs for v2 are not compiled for ARM.

**Comment by stammen** [comment]
@vick-joingame Is this for a Windows RT ARM Tablet? We no longer support these devices as the Angle DLL no longer supports the low level of DirectX on these devices. We support x86 tablets.

**Comment by vick-joingame** [comment]
Yes, this is about Windows RT tablets. Are you sure that ANGLE does not support ARM anymore?  
ANGLE branch still has active ARM targets. As I know, there is no difference for compiling ARM or x86 DLLs. Yeah, some ARM tablets have d3d_feature_level_9_1 and last versions of ANGLE work much slower there, but a lot of ARM tablets have feature_level_9_3 (like Windows Phone).
But anyway, v3-stable branch still have ARM dll, and they work on Windows RT. 
P.S.: I have checked it right now, I copied third party DLLs from v3 branch to 2.2.6 version and now it works as expected. So, probably just somebody put wrong precompiled DLLs (x86 to arm)? Maybe I wrong.
Thanks.
